flink-connector-jdbc/src/main/java/org/apache/flink/connector/jdbc/internal/converter/ClickhouseRowConvert.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.flink.connector.jdbc.internal.converter;
+
+import org.apache.flink.connector.jdbc.converter.AbstractJdbcRowConverter;
+import org.apache.flink.table.types.logical.RowType;
+
+/** */
```

Comment:
well it's better to have some description about the class rather than an empty comment

Reply:
Sorry, there was a slight error in my rebase just now, which caused this to happen. Now this has always been resolved